app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -40,7 +40,7 @@
     }
 
     /**
-     * Generates a KeyPair based on the type selected.
+     * Generates a list with one {@link KeyPair} based on the type selected.
```

Comment:
`* Generates a list of {@link KeyPair}s...`

Even if it is currently always one, in the future this is not necessarily true. If you do want to always give one `KeyPair`, you simply shouldn't return a `List`

In regards to the function after this, that one shouldn't be needed...